woocommerce/woocommerce-gutenberg-products-block
# Fix image link in all products block.

Changes introduced by this PR:
- add the product name as alternative text to the product image
- prevent rendering two image tags when the product is missing an image. (show only the placeholder)
- add a data-label to anchors for products without a product image


Fixes #1654 




#### Accessibility





### How to test the changes in this Pull Request:

1. Add an All product block and make sure you have at least a product without an image
2. Use slow 3G Network setting to see that the placeholder image is loaded until the product image is loaded
3. Reset Network setting and check that product images have alt attributes
4. Check that a product without an image displays a placeholder image, with no alt attribute



### Changelog

> Fix - Improved accessibility of product image links in the products block by using correct aria tags and hiding empty image placeholders.
